scripts/mariadb/playerstats_calculator.sql

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
adding empty file?
Will the store proc definition go here?

Reply:
Yep, will be defining the stored procedure in there.